model/src/main/java/com/microsoft/java/bs/gradle/model/LanguageExtension.java

```diff
@@ -3,17 +3,25 @@
 
 package com.microsoft.java.bs.gradle.model;
 
+import org.gradle.internal.impldep.javax.annotation.Nullable;
```

Comment:
Hey, @hegyibalint do you know if this is fine here? Because I see the Nullable class comes from gradle internal package.

Reply:
I would rather not, as I assume IDE/QA tooling wouldn't pick up this re-packaged class.
If you want to have the annotations, you can depend on it directly just like this:

https://github.com/gradle/gradle/blob/d40c6121608cad99eb818fa698a4daa3eac99aee/build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt#L108

Reply:
I see, thanks @hegyibalint 

@Tanish-Ranjan I think this can be another separate engineer work to enable null check to this project. For now, I think we can just remove those annotation in this PR, and make it clear that it may return null in the javadoc.

Reply:
Done ✅